products/jbrowse-web/src/SessionLoader.ts

```diff
@@ -182,7 +178,7 @@ const SessionLoader = types
      */
     get ready() {
       return Boolean(
-        this.isSessionLoaded && !self.configError && self.pluginsLoaded,
+        this.isSessionLoaded && !self.configError && self.runtimePlugins,
```

Comment:
-this.isSessionLoaded && !self.configError && self.runtimePlugins,
+this.isSessionLoaded && !self.configError && self.runtimePlugins && self.sessionPlugins

this to me seems more obvious, because otherwise i ask myself "why is it not checking sessionPlugins" and have to look at the contents of isSessionLaoded and trace the if statements there to see it checks sessionPlugins being defined there, whereas with this logic, it might be a bit more clear

Reply:
Now uses new `pluginsLoaded` view that checks both runtime and session plugins.